### ISSUE TITLE
Implement some functionality from RevE rebooted- mfkey32 (ReaderAttack) in GUI

### ISSUE THREAD

**Issue by gtpy** [gtpy/ChameleonMini-RevG-GUI#2]
ToDo: Implement ReaderAttack like the one from RevE rebooted

**Comment by iceman1001** [comment]
yeah,  the office revG would benefit greatly from this functionality ;)

**Comment by gtpy** [comment]
@iceman1001 is it the DETECTIONMY command for RevE rebooted to do the mfkey32 attack?
Any idea what depends on it to build it into RevG Firmware?

**Comment by iceman1001** [comment]
yup,  thats the one.   It should be pretty straightforward.

**Comment by gtpy** [comment]
I'll create a separate repo for customizing the firmware with extended features like mkey32 reader attack

**Comment by gtpy** [comment]
@iceman1001 what is the standard behaviour from RevE rebooted in detection mode?
I tried yesterday to implement it in RevG firmware but everytime my cham is freezing.

Can you give me some hints how detectionmy (mf_detection) works at rebooted device?
Was too late yesterday to work more on it ;)

**Comment by iceman1001** [comment]
lets see,  that would be collecting trace data what reader tries authenticate and store it. 
client retrieve it and runs mfkey32_v2 (moebius version)
There is posts about mfkey32  and the code isn't very complex for mfdetection. You might miss what the attack does.

**Comment by gtpy** [comment]
Thanks. If I’ll have a bit more time I’ll check it out.
Too many projects to work on in RL
